### PR TITLE
"setup.bash" didin't work correctly

### PR DESCRIPTION
**Pull request by Mt-is-at-O**
## To do
- remove unnecessary part

### CODE REVIEW

**Comment by m12watanabe1a** [comment]
@Mt-is-at-O Is it possible to change ssh -> https rather than removing the essential packages?

File: build_depends.repos [hunk 1]
```diff
@@ -7,7 +7,3 @@ repositories:
     type: git
     url: https://github.com/HarvestX/h6x_tf_manager.git
     version: humble
-  h6x_tf_manager:
-    type: git
-    url: git@github.com:HarvestX/h6x_tf_manager.git
```
Comment (m12watanabe1a) [inline build_depends.repos:12]: We want https url here

**Review by m12watanabe1a — CHANGES_REQUESTED** [review]

File: build_depends.repos (hunk 1 above)
Comment (Mt-is-at-O) [reply build_depends.repos:12]: https url exists already.
I removed ssh because both ssh and https of tf_manager existed in the file before the change.

Comment (m12watanabe1a) [reply build_depends.repos:12]: Oh sorry, I didn't directly check the file.
Thank you so much!

**Pull request merged**